backend/blueprints/receipts.py

```
@@ -110,9 +120,26 @@ def resolve_image_path(image_path):
-    current_user = get_current_user()
+    # ============================================================================
+    # Authentication & Authorization Setup
+    # ============================================================================
+    # Get Clerk secret key from app config (validated at startup)
+    clerk_secret_key = current_app.config['CLERK_SECRET_KEY']
+
+    # Get frontend origin from environment or default to Vite default port
+    frontend_origin = os.environ.get('FRONTEND_ORIGIN', 'http://localhost:5173')
+
+    sdk = Clerk(bearer_auth=clerk_secret_key)
+    request_state = sdk.authenticate_request(
```

Ruff (0.14.10)

Local variable `request_state` is assigned to but never used

Remove assignment to unused variable `request_state`

(F841)